spec/rails_helper.rb

```diff
@@ -11,4 +12,10 @@
 
 RSpec.configure do |config|
   config.include OAuthHelpers
+  config.include RegistrationsHelpers
+
+  config.around(:each) do |example|
+    DatabaseCleaner.strategy = example.metadata[:type] == :feature ? :seeded : :transaction
```

Comment:
Do you have a use case where we need the `:seeded` strategy? 👀 

If not, I recommend to only keep `:transaction` (as using seeds in a test seems a bit contrary with our conventions to only build the fixtures we need and keep them explicit)